org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hover/html/DefaultEObjectHoverProvider.java

```diff
@@ -535,7 +535,12 @@ public IInformationControlCreator getHoverControlCreator() {
 
 			@Override
 			public Object getInfo() {
-				return getHoverInfo(object, region, null);
+				try {
+					return getHoverInfo(object, region, null);
+				} catch (RuntimeException e) {
+					LOG.warn("Error on computation of hover information. No hover information available for "+ object + " at " + region, e);
```

Comment:
yes but there 3rd party developer might not catch this if it is just logged as warning